src/main/java/cs2103/v15_1j/jimjim/ui/MainViewController.java

```diff
@@ -183,5 +187,18 @@ public void showFatalError(String message){
 			System.exit(0);
 		}
 	}
+	
+	public void addEvent(Event event){
+		boolean clashes = dayPickerPaneController.addEvent(event);
+		
+		if(!clashes){
+			showNotification("\""+event.getName() + "\" has been added.");
+		}
+		else {
+			showNotification("\""+event.getName() + "\" clashes with another event!");
```

Comment:
this function will be executed by AddFeedback when you add an event? If that's the case just move those logic there.